### PR TITLE
UI play trailer

### PR DESCRIPTION
**Pull request by DongHien0896**
Screen Play trailer:
UI: https://edu-redmine.framgia.vn/issues/1289
Get Data: https://edu-redmine.framgia.vn/issues/1293

### CODE REVIEW

**Comment by DongHien0896** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -22,11 +22,11 @@
...
         <activity android:name="com.framgia.hien.moviedb.screen.detail.DetailActivity" />
-
         <activity
             android:name="com.framgia.hien.moviedb.screen.main.MainActivity"
             android:label="@string/app_name" />
+        <activity android:name="com.framgia.hien.moviedb.screen.play.PlayActivity"></activity>
     </application>
-</manifest>
+
+</manifest>
```
Comment (tuannguyen38) [inline app/src/main/AndroidManifest.xml]: need an endline

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -22,11 +22,11 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-
         <activity android:name="com.framgia.hien.moviedb.screen.detail.DetailActivity" />
-
         <activity
             android:name="com.framgia.hien.moviedb.screen.main.MainActivity"
             android:label="@string/app_name" />
+        <activity android:name="com.framgia.hien.moviedb.screen.play.PlayActivity"></activity>
```
Comment (tuannguyen38) [inline app/src/main/AndroidManifest.xml]: only close /> tag

File: app/src/main/java/com/framgia/hien/moviedb/screen/play/PlayViewModel.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        return this.mTrailerAdapter;
+    }
+
+    public void setProgressBar(ProgressBar progressBar) {
+        this.mProgressBar = progressBar;
+    }
+
+    public void getTrailer() {
+        mProgressBar.setVisibility(View.VISIBLE);
+        Disposable disposable = mTrailerRepository.getTrailer(movieObservableField.get().getId(), BuildConfig.API_KEY.toString())
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/hien/moviedb/screen/play/PlayViewModel.java]: this line so long

File: app/src/main/res/layout/item_trailer_recycler.xml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        <TextView
+            android:id="@+id/text_trailer_name"
+            android:layout_width="@dimen/dp_120"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="8dp"
+            android:layout_marginEnd="8dp"
+            android:layout_marginLeft="8dp"
+            android:layout_marginRight="8dp"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="8dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/item_trailer_recycler.xml]: 38 đến 43 hardcode

**Comment by DongHien0896** [comment]
ready

**Pull request merged**